donut/modules/uploads/uploaded_files/pages/2018-2019_Minutes.md

```diff
@@ -0,0 +1,9 @@
+[6 June 2018](6 June 2018)
+
+31 May 2018
+
+23 May 2018: VPSA
+
+      
+
+      
```

Comment:
Let's not put uploads in the git repository

Reply:
Hmm I was thinking of keeping a few around for examples. Also, the documents tab has to be populated with something.

Reply:
Sure, examples are fine, but I don't want the git repo to be the place where actual uploads go. They should just live on the server.

Reply:
Remove all the .md files? This would include the documents pages and all boc/crc related pages...

Reply:
Hmm, if you think the page is *very* unlikely to be edited, I think it's okay to include it. But if the page might change, we probably don't want to have to keep the repo up to date with the copy on the server.